utilities/src/serialize/impls.rs

```diff
@@ -148,7 +148,57 @@ impl_canonical_serialization_uint!(u8);
 impl_canonical_serialization_uint!(u16);
 impl_canonical_serialization_uint!(u32);
 impl_canonical_serialization_uint!(u64);
-impl_canonical_serialization_uint!(usize);
+
+impl CanonicalSerialize for usize {
```

Comment:
I think likely instead of deleting the original serialization path, a `wasm` feature flag might be better suited since the `wasm` feature already exists in SnarkVM utilities.

i.e. something like

```suggestion
#[cfg(feature = "wasm")]
impl_canonical_serialization_uint!(usize);
#[cfg(not(feature = "wasm"))]
impl CanonicalSerialize for usize {
```

Reply:
I think it makes sense to leave for all targets as it also applies to other 32-bit & 128-bit targets.